metabase/metabase
# Adding pagination to All Personal Collections page

Closes https://github.com/metabase/metabase/issues/29177

### Description
This PR makes a few changes to the Other Users Personal Collections page. First, it adds pagination to the users call so that we don't fetch all users at once. Secondly we use regular urls rather than slugified URLs for the links, which removes the requirement to fetch all collections. These 2 changes should dramatically improve the experience of anyone that uses this page in larger instances

### How to verify
1. Navigate to the All Personal Collections page
2. Ensure that pagination is working as expected
3. notice in the network tab that we don't call `/collections` (for this page), and that we grab users 27 at a time.

### Demo

_Upload a demo video or before/after screenshots if sensible or remove the section_

### Checklist
